src/odemis/acq/stream/_static.py

```diff
@@ -374,8 +379,14 @@ def __init__(self, name, data, *args, **kwargs):
         # find positions of each acquisition
         # (float, float, str or None)) -> DataArray: position on SEM + polarization -> data
         self._pos = {}
+
+        # find same ebeam positions of each acquisition, but with different polarization
+        # (float, float)) -> list of DataArrays: position on SEM -> list containing data with different polarization
+        self._pos_polarimetry = {}
```

Comment:
Essentially, `._pos_polarimetry` is identical to `._pos`, but instead of being ebeam_pos + pol -> DataArray, it's ebeam_pos -> pol -> DataArray.
As ensure2DImage returns just a view to the original DataArray, it doesn't take too much extra memory (luckily). However, it feel very redundant. I'd suggest to just modify `_pos` to use that new structure.

Reply:
I think I did it, because otherwise it will be hard to handle it in the two different projections in updateImage. But might have an idea how to solve it.